ibcdfo_pypkg/tox.ini

```diff
@@ -0,0 +1,64 @@
+#
+# Setup and manage python virtual environments that allow users to develop and
+# test the ibcdfo python package.
+#
+
+[tox]
+requires = tox>=4
+env_list = nocoverage
+
+[testenv]
+allowlist_externals = wget
+description = Run the ibcdfo full test suite
+basepython = py3
```

Comment:
Does adding this make a difference?

Reply:
If you are asking about the `wget`, then yes, it's needed. Without it, running `tox -r -e coverage` errors: 
  `coverage: failed with wget is not allowed, use allowlist_externals to allow it`